go/oasis-node/cmd/common/grpc/grpc.go

```diff
@@ -73,6 +77,10 @@ func NewServerLocal(installWrapper bool) (*cmnGrpc.Server, error) {
 func NewClient(cmd *cobra.Command) (*grpc.ClientConn, error) {
 	addr, _ := cmd.Flags().GetString(CfgAddress)
 
+	if len(addr) > 0 && addr[0] == '/' {
```

Comment:
Why not `len(addr) > 0 && !strings.HasPrefix(addr, "unix:")`.  Though at this point, we could automagically try to fix-up the address by prepending the prefix for them if they forget.

Reply:
Just `len(addr) > 0 && !strings.HasPrefix(addr, "unix:")` is not enough, because you can also pass a valid IP. Yeah, I will simply replace the addr, if it smells like a filename.

Reply:
Changed, please review.